k2/csrc/array_ops.cu

```diff
@@ -491,5 +491,12 @@ void RowSplitsToRowIdsOffset(const Array1<int32_t> &row_splits_part,
   RowSplitsToRowIds(row_splits, row_ids_part);
 }
 
+template <>
+Array2<Any> ToContiguous(const Array2<Any> &src) {
+  Array2<Any> ans;
+  FOR_REAL_AND_INT32_TYPES(src.GetDtype(), T,
+                           ans = ToContiguous(src.Specialize<T>()).Generic());
+  return Array2<Any>();  // Silence warning
```

Comment:
Should we return `ans` here? 

Otherwise, the `ans` set inside the macro is never used.